.github/workflows/review.yaml

```
@@ -47,9 +60,12 @@ jobs:
 
     - name: Compile FFmpeg and BMF if not cached
       run: |
-        if [ ! -f "/usr/local/bin/ffmpeg" ]; then
+        if [ ! -d "$(pwd)/bmf/ffmpeg/" ]; then
           echo "FFmpeg not found, starting build..."
-          (cd bmf && git checkout fork_by_oc && sudo scripts/build_ffmpeg.sh nasm yasm x264 x265 opus && ./build.sh )
+          (wget https://ffmpeg.org/releases/ffmpeg-5.1.6.tar.bz2 && tar xjvf ffmpeg-5.1.6.tar.bz2 && 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)